### ISSUE TITLE
cockpit podman causing high load

### ISSUE THREAD

**Issue by daniel-wtd** [cockpit-project/cockpit-podman#562]
Hi all,

i am on fedora 33 with cockpit 229.

- cockpit-podman-24-1.fc33.noarch
- cockpit-ws-229-1.fc33.x86_64

Opening the cockpit page is generating 2 services, which are causing high load after ~20 seconds.

```
└─podman,7454,7454 system service
      ├─{podman},7455,7454
      ├─{podman},7456,7454
      ├─{podman},7457,7454
      ├─{podman},7458,7454
      ├─{podman},7459,7454
      ├─{podman},7461,7454
      ├─{podman},7462,7454
      ├─{podman},7464,7454
      ├─{podman},7496,7454
      ├─{podman},7528,7454
      ├─{podman},7529,7454
      ├─{podman},7530,7454
      └─{podman},7531,7454
```

Dunno, if this coming from some weird behaviour in cockpit-podman or podman itself. Hope you can have a look.

Please don't hesitate to give me a ping, if you need any additional information.

Regards,
Daniel

**Comment by daniel-wtd** [comment]
I have opened a little bit more detailed issue over at podman: https://github.com/containers/podman/issues/8050

**Comment by marusak** [comment]
Thanks!  I also noticed this behavior that when we start podman services they have super high load. Very likely issue in podman itself as it is related to their service taking too many resources.

> I have opened a little bit more detailed issue over at podman: containers/podman#8050

That is the correct place. Closing here.

**Comment by garrett** [comment]
I've upgraded to podman 2.2.0 (in Fedora testing at the moment) and I can confirm it fixes this issue. :tada: 

For reference:
- fix was in https://github.com/containers/podman/pull/7986
- which landed in podman 2.2.0
- it's pending, in Fedora at least, in https://bodhi.fedoraproject.org/updates/FEDORA-2020-7b87810e19

**Comment by daniel-wtd** [comment]
will give it a test asap. Thanks a lot!
